examples/chart/teleport-cluster/charts/teleport-operator/templates/resources.teleport.dev_roles.yaml

```diff
@@ -40,9 +40,7 @@ spec:
                 properties:
                   app_labels:
                     additionalProperties:
```

Comment:
What's the implication of dropping `items` and `type`? Could I do something totally wild like `app_labels: 1` ?

Reply:
Yes, app_labels is still an object but we don't validate anything in it, so you could totally do
```
app_labels:
  key: value
  key2: ["val1", "val2"]
  key3: {"totally":"invalid"}
  key4: 1
```

This is an obviously terrible solution, the alternative is to change the `utils.Strings` marshalling behaviour. @r0mant  thinks we should not touch `utils.Strings` but @marcoandredinis  and @tigrato don't agree. I'll switch back the PR to draft until we reach a consensus

Reply:
Please note that Teleport would reject the invalid ones
So, while it would pass the K8S validations, changes wouldn't be persisted and an error would be returned to the user

Reply:
I'd be wary of messing with `utils.Strings`, but no strong opinions on my part.

Reply:
Yeah I think changing utils.Strings marshaling is asking for more trouble than needed at this point.